Makefile

```diff
@@ -9,6 +9,9 @@ COMPUTE_NODES ?= 2
 
 OCI_BIN ?= docker
 
+USE_STATIC_CHECK ?= true
```

Comment:
I think since you use the `-u` to indicate that it's going to use the static check you can omit having to set it here.